fastlane/Managers/EnvironmentManager.rb

```diff
@@ -1,26 +0,0 @@
-# frozen_string_literal: true
-
-class EnvironmentManager
```

Comment:
I think this change should be another PR referencing https://github.com/nimblehq/ios-templates/issues/330 instead, because removing a helper class is a quite big change 🤔.

Reply:
This class has been unused since we moved to Fastlane Swift so I think we can clean it up in this PR :D

Reply:
Thank you for the clarification 🙏